tools/extra/qpafilter/qpafilter.py

```diff
@@ -456,6 +553,13 @@ def parse_args():
     dump.add_argument('file', type=argparse.FileType('rb'), nargs='?',
                       help='Input blob file')
 
+    dump.add_argument('-o', '--output', default='stdout',
+                      help='Output text file (default=stdout)')
+
```

Comment:
I *think* we can set the type to argparse.FileType('w'), set default to sys.stdout, and set nargs='?'.